cranelift-codegen/meta/src/cdsl/operands.rs

```diff
@@ -202,7 +202,7 @@ impl OperandKindBuilder {
         Self {
             name,
             doc: None,
-            default_member: None,
+            format_field_name: None,
```

Comment:
Along the same lines of a previous comment: to eliminate duplication we could use the `new(...).with_?` pattern or, alternately, keep the current constructor methods but do something like:
```
Self {
  name, fields, ..Default::default()
}
```
(we would have to `#[derive(Default)]` to do this).

Reply:
Yeah, the default approach doesn't work because the name and fields wouldn't be optional, so we'd need a struct just to contain all the default parameters passed to this particular function. I actually tried this and thought it was adding a lot of noise for few value. After this commit, there's only one OperandKind ctor, and the fields are transmitted through helper functions in the appropriate files.